plugins.txt

```diff
@@ -37,7 +37,15 @@ credentials-binding:523.vd859a_4b_122e6
 dark-theme:185.v276b_5a_8966a_e
 datadog:4.0.0
 design-library:104.v492ed3cce3f2
-ec2:1.68
+ec2:incrementals;org.jenkins-ci.plugins;ec2-1.69-rc1489.404b_49d31b_e7;lemeurherve;JENKINS-69304-java-path
```

Comment:
Maybe try dropping the `ec2` prefix from the incremental version to match the `workflow-support` examples in https://github.com/jenkinsci/incrementals-tools#updating-versions-for-jenkins-docker-images.

```suggestion
ec2:incrementals;org.jenkins-ci.plugins;1.69-rc1489.404b_49d31b_e7;lemeurherve;JENKINS-69304-java-path
```